wavefront/server/modules/product_analysis_module/product_analysis_module/controllers/product_anaysis_controllers.py

```
@@ -99,3 +100,46 @@ async def get_product_analysis(
             {'product_analysis': product_analysis_response}
         ),
     )
+
+
+@product_analysis_router.get('/product-analysis/stats/login')
+@inject
+async def get_product_login_stats(
+    request: Request,
+    start_date: date = Query(...),
```

Ruff (0.15.12)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)